lang/en.yml

```diff
@@ -1,383 +1,237 @@
-    DEFAULTSERVERERRORPAGECONTENT: '<p>Sorry, there was a problem with handling your request.</p>'
-    DEFAULTSERVERERRORPAGETITLE: 'Server error'
-  Permissions:
-    CONTENT_CATEGORY: 'Content permissions'
-    PERMISSIONS_CATEGORY: 'Roles and access permissions'
-  RedirectorPage:
-    HASBEENSETUP: 'A redirector page has been set up without anywhere to redirect to.'
-    HEADER: 'This page will redirect users to another page'
-    OTHERURL: 'Other website URL'
-    REDIRECTTO: 'Redirect to'
-    REDIRECTTOEXTERNAL: 'Another website'
-    REDIRECTTOPAGE: 'A page on your website'
-    YOURPAGE: 'Page on your website'
-  RemoveOrphanedPagesTask:
-    BUTTONRUN: Run
-    CHOOSEOPERATION: 'Choose operation:'
-    DELETEWARNING: 'Warning: These operations are not reversible. Please handle with care.'
-    HEADER: 'Remove all orphaned pages task'
-    NONEFOUND: 'No orphans found'
-    NONEREMOVED: 'None removed'
-    OPERATION_REMOVE: 'Remove selected from all stages (WARNING: Will destroy all selected pages from both stage and live)'
-    SELECTALL: 'select all'
-    UNSELECTALL: 'unselect all'
-  SITETREE:
-    VIRTUALPAGEDRAFTWARNING: 'Please publish the linked page in order to publish the virtual page'
-    VIRTUALPAGEWARNING: 'Please choose a linked page and save first in order to publish this page'
-    VIRTUALPAGEWARNINGSETTINGS: 'Please choose a linked page in the main content fields in order to publish'
-  SearchForm:
-    GO: Go
-    SEARCH: Search
-    SearchResults: 'Search Results'
-  SideReport:
-    BROKENFILES: 'Pages with broken files'
-    BROKENREDIRECTORPAGES: 'RedirectorPages pointing to deleted pages'
-    BROKENVIRTUALPAGES: 'VirtualPages pointing to deleted pages'
-    BrokenLinksGroupTitle: 'Broken links reports'
-    ContentGroupTitle: 'Content reports'
-    EMPTYPAGES: 'Pages with no content'
-    LAST2WEEKS: 'Pages edited in the last 2 weeks'
-    ParameterLiveCheckbox: 'Check live site'
-  SilverStripeNavigator:
-    ARCHIVED: Archived
-  SilverStripe\CMS\Controllers\CMSPageAddController:
-    MENUTITLE: 'Add page'
-  SilverStripe\CMS\Controllers\CMSPageEditController:
-    MENUTITLE: 'Edit Page'
-  SilverStripe\CMS\Controllers\CMSPageHistoryController:
-    MENUTITLE: History
-  SilverStripe\CMS\Controllers\CMSPageSettingsController:
-    MENUTITLE: 'Edit Page'
-  SilverStripe\CMS\Controllers\CMSPagesController:
-    MENUTITLE: Pages
-  SilverStripe\CMS\Model\ErrorPage:
-    DESCRIPTION: 'Custom content for different error cases (e.g. "Page not found")'
-    PLURALNAME: 'Error Pages'
-    PLURALS:
-      one: 'An Error Page'
-      other: '{count} Error Pages'
-    SINGULARNAME: 'Error Page'
-  SilverStripe\CMS\Model\RedirectorPage:
-    DESCRIPTION: 'Redirects to an internal page or an external URL'
-    PLURALNAME: 'Redirector Pages'
-    PLURALS:
-      one: 'A Redirector Page'
-      other: '{count} Redirector Pages'
-    SINGULARNAME: 'Redirector Page'
-  SilverStripe\CMS\Model\SiteTree:
```

Comment:
We've lost all the localisations for this key. Can you please restore them @unclecheese ?